README.md

```diff
@@ -10,6 +10,10 @@ This standard sets a quality level for codebases that meets the needs of public
 
 The standard lives at [standard.publiccode.net](https://standard.publiccode.net/), and for an overview of all content see [`index.md`](index.md).
 
+[![The printed Standard for Public Code lying on a table between two hands](https://img.youtube.com/vi/QWt6vB-cipE/mqdefault.jpg)](https://www.youtube.com/watch?v=QWt6vB-cipE)
```

Comment:
This image has a role, perhaps that should be reflected in the alt attribute.

```suggestion
[![Thumbnail for the video on the Standard for Public Code: a printed version lying on a table between two hands](https://img.youtube.com/vi/QWt6vB-cipE/mqdefault.jpg)](https://www.youtube.com/watch?v=QWt6vB-cipE)
```

Reply:
I thought alt-text only should describe what you actually see, not adding semantic meaning of the image to it.

Reply:
The alt text is what displays as an alternative to an image. For people who can see the image it is clearly a 'thumbnail' and they will infer meaning from it. As such it being a thumbnail is an integral part of it's description.

Imagine how someone would describe the elements on the page to you. I'd say 'there is a thumbnail of the video with the standard on it and the title and link underneath."